### PR TITLE
Runbook

### PR DESCRIPTION
**Pull request by huibinshen**
Update runbook instructions, clean the unused datasets.

### CODE REVIEW

**Comment by huibinshen** [comment]
Hi Yuang, please have a look on the instructions, thank you.

File: runbook.md [hunk 1]
```diff
@@ -1,114 +1,150 @@
...
+    ]
+}
+```
+Attach the IAM role `SagemakerAdmin` to the instance. One could do so by doing:
+
+Actions -> Security ->  Modify IAM role -> Select `SagemakerAdmin` -> Update IAM role.
+
+
+### Config AWS CLI
+Before run schedule, we must config the aws cli (only region is needed, they others can be left empty).
```
Comment (Yuang-Deng) [inline runbook.md:42]: I try to config only the region and leave the others empty, but the authentication didn't work

**Review by Yuang-Deng — COMMENTED** [review]
There may be a problem with configuring the AWS CLI

File: runbook.md (hunk 1 above)
Comment (huibinshen) [reply runbook.md:42]: It seems working for Oleksandr, will resolve this.

**Pull request merged**